icu4c/source/data/cldr-icu-readme.txt

```diff
@@ -153,6 +152,17 @@ export CLDR_DIR=$HOME/cldr/trunk
 export ICU4C_DIR=$HOME/icu/icu4c
 export ICU4J_ROOT=$HOME/icu/icu4j
 
+# 1d. Pre-populate your CLDR DTD cache. You need to do this only once.
+
+mkdir ~/.cldrdtd
+cd ~/.cldrdtd
+curl http://www.unicode.org/repos/cldr/trunk/common/dtd/ldml.dtd \
+    -o http___www.unicode.org_repos_cldr_trunk_common_dtd_ldml.dtd
+# WAIT before hitting the server again; it WILL NOT give you a second chance!
+sleep 5
+curl http://www.unicode.org/repos/cldr/trunk/common/dtd/ldmlICU.dtd \
+    -o http___www.unicode.org_repos_cldr_trunk_common_dtd_ldmlICU.dtd
+
```

Comment:
If you are going to do this, you also need at least the ldmlSupplemental.dtd; it may not be used as often as the others but it is used.

Reply:
Done via amend.